src/main/java/com/Futurecollars/Lekcja7/Zadanie7/Training.java

```diff
@@ -0,0 +1,96 @@
+package com.Futurecollars.Lekcja7.Zadanie7;
+
+public class Training {
+    int duration;
+    int burnedCalories;
+    int averageHeartRate;
+
+    public Training(int duration, int burnedCalories, int averageHeartRate) {
+        this.duration = duration;
+        this.burnedCalories = burnedCalories;
+        this.averageHeartRate = averageHeartRate;
+    }
+
+    public int getDuration() {
+        return duration;
+    }
+
+    public int getBurnedCalories() {
+        return burnedCalories;
+    }
+
+    public int getAverageHeartRate() {
+        return averageHeartRate;
+    }
+
+    public int gradeDuration(int duration)
```

Comment:
po co przekazujesz parametr duration skoro masz go w tym obiekcie, przekazałeś go już przez konstruktor i możesz się do niego dostać poprzez this.duration. to samo tyczy się kolejnych metod